src/styles/blocks/logo.css

```
@@ -40,12 +40,24 @@
 
 .logo__title {
     margin: 0;
-    font-weight: bold;
+    font-family: 'Dewi', sans-serif;
+    font-weight: 700;
     font-size: 20px;
     line-height: 2;
     letter-spacing: -0.03em;
 }
 
+.logo__title:hover,
+.logo__title:focus {
+    font-family: 'Dewi Expanded', sans-serif;
+    font-weight: 600;
```

```suggestion
    font-weight: bold;
```

Почему `600`, а не `bold`? См. выше.